### PR TITLE
Add note about writeback mode in disk cache

### LINKED ISSUE #627: Disk Device Cache mode update
Support for `writeback` caching was added in this PR: https://github.com/kubevirt/kubevirt/pull/6968

However the current docs do not reflect this: https://kubevirt.io/user-guide/virtual_machines/disks_and_volumes/#disk-device-cache

### PR DESCRIPTION
**Pull request by cwilkers**
Add to documentation that writeback is a valid cache mode for disks

Fixes #627

I did some research on the feature, but would appreciate sanity-check from some of the devs that I'm not misleading anybody here.

### CODE REVIEW

**Comment by cwilkers** [comment]
@ChandonPierre @adithyaakrishna @vladikr Can you PTAL? Thanks!

File: docs/virtual_machines/disks_and_volumes.md [hunk 1]
```diff
@@ -1451,14 +1451,17 @@ allocate 4 queues (to match the number of CPUs requested).
...
 -   `none` I/O from the guest is not cached on the host. Use this option
     for guests with large I/O requirements. This option is generally the
     best choice.
 
+-   `writeback` I/O from the guest is cached on the host and written through
+    to the physical media when the guest OS issues a flush.
+
 -   `writethrough` I/O from the guest is cached on the host but written
-    through to the physical medium.
+    through to the physical medium on each write.
```
Comment (vladikr) [inline docs/virtual_machines/disks_and_volumes.md:1464]: I'm not 100% sure how to phrase it best. Perhaps we should say that the data is stored in the page cache on the host and writes are reported as completed to the guest only when the data has been committed to the storage device.

@alicefr what do you think?

Comment (alicefr) [reply docs/virtual_machines/disks_and_volumes.md:1464]: @vladikr yes, it sounds a more correct formulation to me. Here, we are missing that the writes are completed when they are done on the storage device.

Comment (cwilkers) [reply docs/virtual_machines/disks_and_volumes.md]: I updated the wording, is this more correct?

Comment (alicefr) [reply docs/virtual_machines/disks_and_volumes.md]: Yes, sounds good to me

**Comment by vladikr** [comment]
Looks good, thanks!
/lgtm

**Comment by cwilkers** [comment]
Thanks!
Since I've got two lgtms, I'll approve my own PR:
/approve

**Pull request merged**